updatecli/updatecli.d/updatecilium.yaml

```diff
@@ -27,6 +28,7 @@ targets:
       environments:
         - name: CILIUM_VERSION
           value: '{{ source "cilium" }}'
+        - name: PATH
```

Comment:
don't you need to specify the PATH value? It just propagates it?

Reply:
If not specified it uses the one already configured. The shell updatecli action will not propagate the environment variable by default